src/language/mongoDBService.ts

```diff
@@ -101,15 +109,23 @@ export default class MongoDBService {
     try {
       // Get database names for the current connection.
       const databases = await this._getDatabases();
-
       // Create and cache database completion items.
       this._cacheDatabaseCompletionItems(databases);
-      return Promise.resolve(true);
     } catch (error) {
       this._connection.console.error(
         `LS get databases error: ${util.inspect(error)}`
       );
-      return Promise.resolve(false);
+    }
+
+    try {
+      // Get log names for the admin database.
+      const logNames = await this._getLogs();
+      // Cache log names for diagnostic fixes.
+      this._cacheLogNames(logNames);
+    } catch (error) {
+      this._connection.console.error(
+        `LS get logs error: ${util.inspect(error)}`
+      );
```

Comment:
Fwiw, I don’t really expect people to try to run `show log ...` in the VSCode extension code, I don’t think we need to put in a lot of effort to support it. I know the code’s here already written, so feel free to just resolve this, but yeah, for 99.9 % of users the slowdown from the extra `getLog` command that’s being run for every connection here will not be useful to them.

Reply:
I kept log, but considering the fact that dbs also can have new names, I've rewritten the whole block, so I parse the string to find a database name and log type, or whatever can come later that has a replacement value and use it when a build a fix string.